src/net_processing.cpp

```diff
@@ -1373,7 +1375,11 @@ static void RelayAddress(const CAddress& addr, bool fReachable, const CConnman&
     // Use deterministic randomness to send to the same nodes for 24 hours
     // at a time so the m_addr_knowns of the chosen nodes prevent repeats
     uint64_t hashAddr = addr.GetHash();
-    const CSipHasher hasher = connman.GetDeterministicRandomizer(RANDOMIZER_ID_ADDRESS_RELAY).Write(hashAddr << 32).Write((GetTime() + hashAddr) / (24 * 60 * 60));
+
+    std::chrono::seconds rotate_interval = std::chrono::duration_cast<std::chrono::seconds>(ROTATE_ADDR_RELAY_DEST_INTERVAL);
+    std::chrono::seconds current_time = GetTime<std::chrono::seconds>();
+    uint64_t time_seed = current_time.count() / rotate_interval.count();
```

Comment:
```suggestion
    const uint64_t time_seed {GetTime<std::chrono::seconds>().count() / std::chrono::seconds{ROTATE_ADDR_RELAY_DEST_INTERVAL}.count()};
```

Everything in one line with compile-time checks enabled would look like this, btw

Reply:
can we keep it to 2 lines? I think what you suggest is hard to read.